mlcomp/tests/test_mlcomp_python.py

```diff
@@ -12,6 +12,7 @@
 
 import parallelm.pipeline.json_fields as json_fields
 
+REFLEXCOMMON_PATH = "../../reflex-common"
```

Comment:
Move this to constants file within this repo

Reply:
This is related to tests only, not to the mlcomp package.
Probably I'm not going to move it anythere fo now.